src/richie/apps/core/templates/richie/base.html

```diff
@@ -81,47 +81,54 @@
-                                        <li class="topbar__item topbar__item--login richie-react richie-react--user-login" data-props='{"profileUrls": {{ AUTHENTICATION.profile_urls }}}'></li>
-                                    {% endif %}
-                                {% endblock userlogin %}
-                                {% block topbar_contact %}
-                                <li class="topbar__item topbar__item--cta">
-                                    <a href="#">{% trans "Contact us" %}</a>
-                                </li>
-                                {% endblock topbar_contact %}
-                            </ul>
+                                <button
+                                    class="topbar__hamburger"
+                                    data-target="main-menu"
+                                    aria-label="{% trans "Menu" %}"
+                                    aria-expanded="false"
+                                >&#8801;</button>
+                            </div>
+                            <div class="topbar__menu topbar__menu--aside">
+                                {% block topbar_searchbar %}
+                                <div class="topbar__search richie-react richie-react--root-search-suggest-field"
+                                    data-props='{"courseSearchPageUrl": "{% page_url 'courses' %}"}'></div>
+                                {% endblock topbar_searchbar %}
+                                <ul class="topbar__list topbar__list--controls">
+                                    {% block userlogin %}
+                                        {% if AUTHENTICATION %}
+                                            <li class="topbar__item topbar__item--login richie-react richie-react--user-login" data-props='{"profileUrls": {{ AUTHENTICATION.profile_urls }}}'></li>
+                                        {% endif %}
+                                    {% endblock userlogin %}
+                                    {% block topbar_contact %}
+                                    <li class="topbar__item topbar__item--cta">
+                                        <a href="#">{% trans "Contact us" %}</a>
+                                    </li>
+                                    {% endblock topbar_contact %}
+                                </ul>
 
-                            {% language_chooser "menu/language_menu.html" %}
-                        </div>
+                                {% language_chooser "menu/language_menu.html" %}
+                            </div>
+                        </header>
+
+                        <nav class="topbar__menu">
+                            <ul class="topbar__list">
+                                {% show_menu 0 100 0 0 "menu/header_menu.html" %}
+                            </ul>
+                        </nav>
                     </div>
-                </nav>
-            </header>
+                </div>
+            </div>
```

Comment:
most of the header/nav changes here are just blocks that have been reordered, don't hesitate to check this PR with hidden whitespace changes.

Reply:
Had no idea about this NVDA issue. I always encapsulate <nav> semantically in the element it should be displayed. Thanks for this note.